### ISSUE TITLE
BTE returning "unknown" on Workflow C query

### ISSUE THREAD

**Issue by vgardner-renci** [NCATSTranslator/minihackathons#51]
![image](https://user-images.githubusercontent.com/83781286/123322917-d8454000-d502-11eb-8b9d-8b52ef293813.png)

0fd681f9-4b08-43d4-b7a3-cf6e561d49af

**Comment by rtroper** [comment]
Not sure if this is in scope (for BTE) for the December demo.

**Comment by andrewsu** [comment]
This is a "branched" query topology that I think BTE does not handle gracefully. Working on it...

```
{
    "message": {
        "query_graph": {
            "nodes": {
                "n0": {
                    "categories": ["biolink:Gene"]
                },
                "n1": {
                    "ids": ["CHEBI:45783"],
                    "categories": ["biolink:ChemicalSubstance"]
                },
                "n2": {
                    "ids": ["MONDO:0005301"],
                    "categories": ["biolink:Disease"]
                },
                "n3": {
                    "categories": ["biolink:ChemicalSubstance"]
                }
            },
            "edges": {
                "e01": {
                    "subject": "n0",
                    "object": "n1",
                    "predicates": ["biolink:related_to"]
                },
                "e02": {
                    "subject": "n0",
                    "object": "n2",
                    "predicates": ["biolink:related_to"]
                },
                "e03": {
                    "subject": "n0",
                    "object": "n3",
                    "predicates": ["biolink:related_to"]
                }
            }
        }
    }
}
```

**Comment by vgardner-renci** [comment]
https://arax.ncats.io/?source=ARS&id=bc7d0408-402c-480e-ad36-da57145834fc

![image](https://user-images.githubusercontent.com/83781286/127528330-15931c97-3326-44e0-b7e6-db768ac81587.png)


```{
  "callback": "http://0.0.0.0:8000/ars/api/messages/138d3a82-3496-4ac6-b9c5-4467b26eac6f",
  "message": {
    "query_graph": {
      "edges": {
        "e01": {
          "object": "n1",
          "predicates": [
            "biolink:related_to"
          ],
          "subject": "n0"
        },
        "e02": {
          "object": "n2",
          "predicates": [
            "biolink:related_to"
          ],
          "subject": "n0"
        },
        "e03": {
          "object": "n3",
          "predicates": [
            "biolink:related_to"
          ],
          "subject": "n0"
        }
      },
      "nodes": {
        "n0": {
          "categories": [
            "biolink:Gene"
          ]
        },
        "n1": {
          "categories": [
            "biolink:ChemicalSubstance"
          ],
          "ids": [
            "CHEBI:45783"
          ]
        },
        "n2": {
          "categories": [
            "biolink:Disease"
          ],
          "ids": [
            "MONDO:0005301"
          ]
        },
        "n3": {
          "categories": [
            "biolink:ChemicalSubstance"
          ]
        }
      }
    }
  },
  "validation_result": {
    "message": "",
    "size": "2 kB",
    "status": "PASS",
    "version": "1.1.1"
  },
  "araxui_provider": "ARS",
  "araxui_response": "138d3a82-3496-4ac6-b9c5-4467b26eac6f"
}

**Comment by andrewsu** [comment]
Branched queries like this _may_ be addressed by biothings/BioThings_Explorer_TRAPI#112 -- we'll have to test once that's complete.
